src/gluonts/core/component.py

```diff
@@ -496,17 +493,17 @@ def num_gpus(refresh=False):
     return NUM_GPUS
 
 
+@functools.lru_cache()
```

Comment:
Could this be an issue, or is `mx.context.cpu()` always the same?

Reply:
Yes, should be ok.
This is to avoid logging things multiple times?

Reply:
Yes.